app/src/main/res/layout/fragment_recommendations_list.xml

```diff
@@ -5,19 +5,20 @@
     android:layout_height="match_parent"
     app:layout_behavior="@string/appbar_scrolling_view_behavior">
 
-    <android.support.v4.widget.SwipeRefreshLayout
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        android:id="@+id/swipeContainer"
+    <android.support.v7.widget.RecyclerView
+        android:id="@+id/rvRecommendations"
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
+        android:background="#eee"
+        android:clipToPadding="false"
+        android:layout_height="match_parent"/>
 
-        <android.support.v7.widget.RecyclerView
-            android:id="@+id/rvRecommendations"
-            android:layout_width="match_parent"
-            android:background="#eee"
-            android:clipToPadding="false"
-            android:layout_height="match_parent"/>
-
-    </android.support.v4.widget.SwipeRefreshLayout>
+    <ImageView
```

Comment:
We might be able to make a custom progress bar that wraps up the animation and all that. I can look into making one.